packages/terra-button/src/Button.jsx

```diff
@@ -244,7 +244,7 @@ class Button extends React.Component {
         { block: isBlock },
         { compact: isCompact },
         { 'is-active': this.state.active && !isDisabled },
-        { 'is-focused': this.state.focused && !isDisabled },
+        { 'is-focused': this.state.focused && !isDisabled && document.activeElement === this.buttonRef },
```

Comment:
should we set the focused state to false when button is set to disabled ?

Reply:
Updated 8f4df52